routes/api_v2.php

```diff
@@ -392,7 +390,7 @@
     Route::get('/projects', ViewMyProjectsController::class);
 });
 
-Route::post('/users/resend', [AuthController::class, 'resendByEmail']);
+Route::post('/users/resend', [AuthController::class, 'resendByEmail'])->withoutMiddleware('auth:service-api-key,api');
```

Comment:
This was the first bug found by the resurrected test suite. This endpoint currently doesn't work in production, but I as far as I know nobody has reported it yet.